CONTRIBUTING.md

```diff
@@ -2,44 +2,58 @@
 
 ## Contributing Words
 
 ### Version 1: Issues
 
-If you know a word that should be defined, but – for whatever reason – don’t feel confident writing the definition yourself (or maybe you disagree with an existing defintion), please open an issue. We can then collectively figure out how to best get this word defined.
+If you know a word that should be defined, but – for whatever reason – don’t feel confident writing the definition yourself (or maybe you disagree with an existing defintion), please open an [issue](https://github.com/tatianamac/selfdefined/issues/new). We can then collectively figure out how to best get this word defined.
 
 An issue might also be appropriate if you are unsure about some nuances of a definition you want to define.
 
 ### Version 2: Pull Requests
 
-You got a definition and want to define it. That’s awesome. Let’s go. Currently you need to fork this repository and open a PR out of your fork. If that sounds like gibberish to you, Chaser Pettit wrote a [great introduction to the workflow](https://gist.github.com/Chaser324/ce0505fbed06b947d962).
+You got a definition and want to define it. That’s awesome. Let’s go. Currently, you need to fork this repository and open a PR out of your fork. If that sounds like gibberish to you, Chaser Pettit wrote a [great introduction to the workflow](https://gist.github.com/Chaser324/ce0505fbed06b947d962).
 
 #### Working on Words
 
-After you’ve forked the repo, you will need to write a definition. Good times. Hopefully. Always remember, if you are unsure you can first open an issue or create a PR, include «WIP» in the title and ask for early feedback.
+After you’ve forked the repo, you will need to write a definition. Good times. Hopefully. Always remember, if you are unsure, you can first open an issue or create a PR, include «WIP» in the title, and ask for early feedback.
 
-All definitions are stored in the [definitions folder](11ty/definitions/). Create a file named `word.md` in it. `word` should be the name of the word you are defining. Not `word`. Except you are defining «word». In this case it should be `word`. For this tutorial I will take the word «Obsessive Compulsive Disorder», as it allows me to show some of the feature. That means I would create the file `obsessive-compulsive-disorder.md` (as it is a defined word, you can have a look at [the finished definition](11ty/definitions/obsessive-compulsive-disorder.md)).
+- Step 1: Create a file named `word.md` in in the [definitions folder](11ty/definitions/) where we store all definitions.
 
-All definitions consist of two parts: Front Matter and content. Front Matter is technical bubble for some structured data about your content. [You can find the complete documentation on our website](https://www.selfdefined.app/documentation/front-matter/). Here’s a quick run down:
+  `word` should be the name of the word you are defining. Not `word`. Except you are defining «word». In this case it should be `word`. 
```

Comment:
I think this is funny but I wonder upon second read if it could be kind of confusing?

Reply:
That’s my personal preference for documentation that does not take itself to seriously shining through. 🙈

Agree on the point that it might be confusing, though.

Reply:
I liked the humor as well, but it might confuse someone, that's true. I rewrote this paragraph.